src/App.js

```diff
@@ -1,12 +1,62 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import PokemonCard from "./assets/Components/PokemonCard";
+import logo from "./assets/Logo/Pokemon-logo.png";
+import axios from "axios";
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemon, setPokemon] = useState([])
```

Comment:
Zodra de setter aangeroepen wordt dan is het geen array mee. Ik zou dan de default waarde `undefined` maken
```suggestion
    const [pokemon, setPokemon] = useState()
```